using-webhooks/client/ios/App (ObjC)/CheckoutViewController.m

```diff
@@ -96,7 +96,8 @@ - (void)startCheckout {
         }
         else {
             NSLog(@"Created PaymentIntent");
-            self.paymentIntentClientSecret = json[@"clientSecret"];
+            NSDictionary *dataDict =[NSJSONSerialization JSONObjectWithData:data options:0 error:nil];
```

Comment:
Ah, thanks so much for fixing this @kristiehuang!

We should also create this dictionary on line 93, and check `dataDict[@"publishableKey"] == nil` in the `if` statement below instead of `json[@"publishableKey"]`.  I'm happy to make that change in a separate commit if you'd like.

Reply:
Yep, just made that change!

Reply:
🙏